src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -186,6 +187,10 @@ export class MeetingManager implements AudioVideoObserver {
       this.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
     }
 
+    if (reconnectTimeoutMs) {
```

Comment:
Do not have to this right? You are already de-structuring from the `this.meetingManagerConfig` above.

Reply:
Yes no need to do it.